.github/workflows/release.yml

```diff
@@ -25,3 +25,39 @@ jobs:
         run: |
           python3 -m build --sdist --wheel .
           twine upload dist/*
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv
+          pipenv install --dev --system --skip-lock
+          pip install https://github.com/Toloka/stubmaker/archive/refs/heads/main.zip
+      - name: Checkout docs repository
+        uses: actions/checkout@v3
+        with:
+          repository: 'Toloka/docs'
+          path: 'docs-repo'
+      - name: Generate markdowns
+        run: |
+          python ./misc/make_markdowns.py --module-root crowdkit --src-root crowdkit --output-dir docs/reference \
```

Comment:
Let's `chmod +x misc/make_markdowns.py`, add a shebang there for Python 3 (`#!/usr/bin/env python3`), and then simply run it as `./misc/make_markdowns.py`.